### PR TITLE
Deploy package to anaconda and pypi

### PR DESCRIPTION
**Pull request by Longbowman**
I am writing github actions for deploying master code to anaconda channel and pypi to install it using pip and anaconda. 

Justin could you chat with me on these matters?

### CODE REVIEW

**Comment by Longbowman** [comment]
Hi Justin
I have uploaded a package to anaconda. Could you please install it to anaconda and test?

**Comment by Longbowman** [comment]
Justin. 
Now even import test doesn't fails so please review whenever you can.

**Comment by Longbowman** [comment]
Hi Justin
Could you test this today or tomorrow?

**Comment by Longbowman** [comment]
Hi Justin. 
Did you get time to test it by chance?

**Comment by Longbowman** [comment]
Hi Justin 
may 4th you told me that you can work on it tomorrow. Could you at least set an appointment?

**Comment by Longbowman** [comment]
Justin you can install it and check version at least.

**Comment by Longbowman** [comment]
Hi Justin 
Please review this result 
https://pastebin.com/wi9Ws0A4

File: .github/workflows/pythonpublishpypilinux.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+on:
+  release:
+    types: [created]
+    
+#on:
+  #pull_request:
+  #push:
+    #branches:         # array of glob patterns matching against refs/heads. Optional; defaults to all
+      #- master          # triggers on pushes that contain changes in master
```
Comment (justinrporter) [inline .github/workflows/pythonpublishpypilinux.yml:11]: Can we remove this commented code?

File: .github/workflows/pythonpublishpypimacos36.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+on:
+  release:
+    types: [created]
+    
+#on:
+  #pull_request:
+  #push:
+    #branches:         # array of glob patterns matching against refs/heads. Optional; defaults to all
+      #- master          # triggers on pushes that contain changes in master
```
Comment (justinrporter) [inline .github/workflows/pythonpublishpypimacos36.yml:11]: Let's cut the comments from here also

File: .github/workflows/pythonpublishzminiconda37.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    steps:
+    - uses: actions/checkout@master
+    - name: Install essential libraries 
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libmpich12 libmpich-dev build-essential ca-certificates 
+    #- name: Sleep for 5 minutes
+      #uses: jakejarvis/wait-action@master
+      #with:
+        #time: '5m'
```
Comment (justinrporter) [inline .github/workflows/pythonpublishzminiconda37.yml:25]: more stray comments.

File: meta.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    Enspara is primarily a python library, but it also contains a few applications for very common Markov-state model related tasks.
+
+    MSM estimation and manipulation
+    Turn-key applications for clustering and analysis
+    Transition path theory
+    Information theory
+    Ragged ndarrays
+  dev_url: https://github.com/bowman-lab/enspara
+  doc_url: https://pypi.org/project/enspara/
+  doc_source_url: https://enspara.readthedocs.io/en/latest/index.html
```
Comment (justinrporter) [inline meta.yaml:75]: I think the `doc_url` should be readthedocs and `doc_source_url` should be the `docs/` in this repository.

**Comment by Longbowman** [comment]
Justin
Do you require any more changes until I leave this repository?

**Comment by Longbowman** [comment]
Justin could you merge this PR so that I leave the repository?

**Pull request merged**